Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,8 @@
 import React from 'react';
 import './Card.css';
+
+function CardCBanner() {
+    return <img src="https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png" className="card-banner"></img>;
```

Comment:
*img* elements are self-closing tags in JSX. You can also add an *alt* attribute for better accessibility and to default to in case the image fails to load.
https://facebook.github.io/create-react-app/docs/adding-images-fonts-and-files